tests/driver.c

```diff
@@ -151,6 +151,24 @@ void test_find_transactions_obj_by_tag(void) {
   free(json_result);
 }
 
+void test_receive_mam_message(void) {
+  char* json_result;
+  double sum = 0;
+
+  for (size_t count = 0; count < TEST_COUNT; count++) {
+    clock_gettime(CLOCK_REALTIME, &start_time);
```

Comment:
Shall we write time utility functions to simply these test cases? Then, we can avoid the repeated calls.

Reply:
I think it's good to have.  I'll open an issue for this.